### PR TITLE
Enhance docstring+fix backward compatiblity manipulation

### PR DESCRIPTION
**Pull request by lorenzovecchietti**

### CODE REVIEW

**Review by Samuelopez-ansys — DISMISSED** [review]
lgtm

File: pyaedt/modules/MeshIcepak.py [hunk 1]
```diff
@@ -408,6 +409,12 @@ def parts(self, parts):
 
 
 class MeshSettings(object):
+    """
+    Class for managing mesh settings.
+    It can be used like a dictionary. Available keys change according
```
Comment (PipKat) [inline pyaedt/modules/MeshIcepak.py:414]:
```suggestion
    
    This class can b
    Blank line needs to separate the brief description from any additional information.

File: pyaedt/modules/MeshIcepak.py [hunk 2]
```diff
@@ -464,14 +471,14 @@ def _dim_arg(self, value):
...
+    def parse_settings_as_args(self):
         """
         Parse mesh region settings.
 
         Returns
         -------
-        dict
-            List of strings containing all the parts that must be included in the subregion.
+        List
+            Arguments to be passed to native APIs.
```
Comment (PipKat) [inline pyaedt/modules/MeshIcepak.py:481]:
```suggestion
            Arguments to pass to native APIs.
```
Use present tense.

File: pyaedt/modules/MeshIcepak.py (hunk 1 above)
Comment (lorenzovecchietti) [reply pyaedt/modules/MeshIcepak.py:414]:
```suggestion
    
    It can be used like a dictionary. Available keys change according
```

**Pull request merged**